interface/src/index.html

```diff
@@ -98,6 +103,19 @@
             <profile-button ng-if="userObj"></profile-button>
           </ul>
         </div>
+
+        <!-- Current machine learning model
+        <div class="row">
+          <div class="col-sm-1"></div>
+          <div class="col-sm-11">
+            <span>Current Model: </span>
+            <span class="text-danger" ng-bind="modelInfo.title || 'Not Set'">
+            </span>
+            <span>(Click <a href="#/home">Home</a> to change)</span>
+          </div>
+        </div>
+        End of machine learning model info part
+        -->
```

Comment:
@mhuyck: You are right. I forgot to remove these comments. Since I have already merged the PR, I am going to file an issue to remove these lines later.